src/main.rs

```diff
@@ -246,9 +246,31 @@ async fn run(miniserve_config: MiniserveConfig) -> Result<(), ContextualError> {
 
     println!(
         "Availabe at (non-exhaustive list):\n    {}\n",
-        display_urls.join("\n    "),
+        display_urls
+            .iter()
+            .map(|url| Color::Green.paint(url).bold().to_string())
+            .collect::<Vec<_>>()
+            .join("\n    "),
     );
 
+    // print QR code to terminal
+    if miniserve_config.show_qrcode && atty::is(atty::Stream::Stdout) {
```

Comment:
Are we perhaps going to use a new CLI flag for this? As it stands, this one is used for presenting the QR code in the web UI. Not sure about this.

Reply:
I think it is too much to have two different options for QR code. though I have no strong opinion on this. I'd add another option if you want.

Reply:
Eh let's leave it like it is until someone complains.